conf/application.conf

```diff
@@ -50,29 +54,158 @@ play.i18n.langs = ["en"]
-  uk.gov.hmrc.individualssavingsincomeapi.controllers.MicroserviceHelloWorldController = {
-    needsAuth = false
-    needsLogging = false
-    needsAuditing = false
+# You can disable evolutions if needed
+# evolutionplugin=disabled
+
+# Api related config
+api {
+  # The status of the version of the API for the API Platform.
+  1.0 {
+    status = "BETA"
+    endpoints.enabled = true
+  }
+
+  confidence-level-check {
+    confidence-level = 250
+    definition.enabled = true
+    auth-validation.enabled = true
   }
+
+  # The context which the API will have via the API Platform http://API_GATEWAY/{api.gateway.context}/
+  gateway.context = "individuals/savings-income"
+}
+
+bootstrap.http.headersAllowlist = [ "Accept", "Gov-Test-Scenario", "Content-Type", "Location", "X-Request-Timestamp", "X-Session-Id" ]
+
+internalServiceHostPatterns = [ "localhost" ]
+
+# Metrics plugin settings - graphite reporting is configured on a per env basis
+metrics {
+  name = ${appName}
+  rateUnit = SECONDS
+  durationUnit = SECONDS
+  showSamples = true
+  jvm = true
+  enabled = true
 }
 
 # Microservice specific config
 
+auditing {
+  enabled = true
+  consumer {
+    baseUri {
+      host = localhost
+      port = 8100
+    }
+  }
+}
+
+feature-switch {
+  opw.enabled = true
+  allowTemporalValidationSuspension.enabled = true
+  postCessationReceipts.enabled = "true"
+  passDeleteIntentHeader.enabled = true
+}
 
 microservice {
+  metrics {
+    graphite {
+      host = graphite
+      port = 2003
+      prefix = play.${appName}.
+      enabled = false
+    }
+  }
+
   services {
+
     auth {
       host = localhost
       port = 8500
     }
+
+    mtd-id-lookup {
+      host = localhost
+      port = 9769
+    }
+
+    des {
+      host = 127.0.0.1
+      port = 9772
+      env = Prod
+      token = ABCD1234
+      environmentHeaders = ["Accept", "Gov-Test-Scenario", "Content-Type", "Location", "X-Request-Timestamp", "X-Session-Id", "X-Request-Id"]
+    }
+
+    ifs {
+      host = 127.0.0.1
+      port = 9772
+      env = Prod
+      token = ABCD1234
+      environmentHeaders = ["Accept", "Gov-Test-Scenario", "Content-Type", "Location", "X-Request-Timestamp", "X-Session-Id", "X-Request-Id"]
+    }
+
+    tys-ifs {
+      host = 127.0.0.1
+      port = 9772
+      env = Prod
+      token = ABCD1234
+      environmentHeaders = ["Accept", "Gov-Test-Scenario", "Content-Type", "Location", "X-Request-Timestamp", "X-Session-Id", "X-Request-Id"]
+    }
+
+    release6 {
+      host = 127.0.0.1
+      port = 9772
+      env = Prod
+      token = ABCD1234
+      environmentHeaders = ["Accept", "Gov-Test-Scenario", "Content-Type", "Location", "X-Request-Timestamp", "X-Session-Id", "X-Request-Id"]
+    }
+
+    api1661 {
+      host = 127.0.0.1
+      port = 9772
+      env = Prod
+      token = ABCD1234
+      environmentHeaders = ["Accept", "Gov-Test-Scenario", "Content-Type", "Location", "X-Request-Timestamp", "X-Session-Id", "X-Request-Id"]
+    }
```

Comment:
These aren't used. (They were for other income-received endpoints.) Can we remove them along with common (base connector trait) code that uses them. Not sure if they're in environment config but it would be simpler if they weren't.